test/system/uniswap-v2.test.js

```diff
@@ -96,17 +118,17 @@ describeFn("System -- swap signer bonds on UniswapV2", () => {
     before(async () => {
       tx = await signerBondsUniswapV2
         .connect(thirdParty)
-        .swapSignerBondsOnUniswapV2(ethers.utils.parseEther("10"), {
-          gasLimit: 185000,
+        .swapSignerBonds(riskManagerV1.address, ethers.utils.parseEther("10"), {
+          gasLimit: 200000,
         })
     })
 
     it(
-      "should take the swapped amount from the swap strategy " +
+      "should take the swapped amount from the risk manager " +
         "contract balance",
```

Comment:
This test name should be short enough to keep it in one line.

Reply:
https://github.com/keep-network/coverage-pools/pull/88/commits/91d89730db307314c606713e113a4d42294f694a